css/import.scss

```
@@ -31,6 +31,12 @@
 			text-align: center;
 		}
 
+		.import-modal__actions {
+			> button {
+				display: inline-block !important;
+			}
```

```suggestion
			display: flex;
			gap: 5px;
```

Why not use flex? The button selector doesn't have to be hard coded this way.